fixieai/cli/validators.py

```diff
@@ -0,0 +1,17 @@
+import re
+
+AGENT_ID_PATTERN = re.compile(r"^\w+$")
```

Comment:
It would be better to use the existing `validators` package for this, in this case `validators.slug`. https://validators.readthedocs.io/en/latest/#module-validators.slug

(The agent handle should be a slug so this is the appropriate check.)

Reply:
I was hesitant to install a new package for this, but went ahead and did it. It seems to be lean package.